frontend/packages/core/src/Theme/theme.tsx

```diff
@@ -20,6 +20,9 @@ const createTheme = (variant: ThemeVariant): MuiTheme => {
   return createMuiTheme({
     colors: clutchColors(variant),
     palette: palette(variant),
+    // `8` is the default scaling factor in MUI, we are setting it again to make it explicit
+    // https://v5.mui.com/material-ui/customization/spacing/
+    spacing: 8,
```

Comment:
Thanks for the usage example 🙌
So, just a thought, would it be a good idea to include the spacing options as [properties](https://mui.com/material-ui/customization/theming/#custom-variables) of the theme object?

Reply:
I was going with that approach at first, but after discovering the `theme.spacing` method, I wanted to take advantage of what it could be done with it, like the [multiple arity](https://v5.mui.com/material-ui/customization/spacing/#multiple-arity).

Reply:
Gotcha, I think they can coexist, something like `theme.spacing(theme.standardSpacing.medium)` and in that `standardSpacing` you pretty much do the same you already have
I can see the naming and having to access the properties in `theme` twice a bit cumbersome 😅 though
What I do like about this is centralising everything in `theme`

Reply:
Agreed, I think centralizing it would improve its DevEx. 

Maybe we could add a new function to be used instead of `spacing()`, or come up with a different naming strategy for the spacing to make it concise?

Reply:
Something like `theme.clutchSpacing.medium`?, it is still long 😆 but yeah, that sounds easy to use

Reply:
I think the `clutch` prefix makes sense, so I turned that into an object in b40fc87. It ended up like:

```
theme.spacing(theme.clutch.spacing.md);
```

I was aiming to create something that only relies on the variant being the functions themselves, but that would require ugly workarounds, like creating the theme twice. Also, using `this` was unstable, even with arrow functions.

Should we settle for this?

cc @jdslaugh

Reply:
Yeah I think this is a good way to go for the time being.